### PR TITLE
Moved exercise files

### PR DESCRIPTION
**Pull request by vulcanpanther**

### CODE REVIEW

File: Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+1. Install [Python Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+
+
+## Set up your Azure resources
+
+To save you time, select this Azure ARM template to create resources you'll need later in the exercise.
+
+### Deploy a pre-built ARM template
+
+1. [![Deploy to Azure.](../media/04-media//deploy-azure.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoftLearning%2Fmslearn-doc-intelligence%2Fmain%2Fcognitive-search%2Fazuredeploy.json) select this link to create your starting resources. You might need to copy and paste the [direct link](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoftLearning%2Fmslearn-doc-intelligence%2Fmain%2Fcognitive-search%2Fazuredeploy.json) into your search bar.
```
Comment (ivorb) [inline Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md:28]: extra `/`?

File: Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md [hunk 2]
```diff
@@ -0,0 +1,531 @@
+You've built a search solution and now want to add Azure AI Services for language enrichments to your indexes.
+
+:::image type="content" source="../media/04-media//exercise-overview.png" alt-text="A diagram showing the components of this exercise. Azure Cognitive Search, Azure AI Language, and a Function App." border="false":::
```
Comment (ivorb) [inline Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md:3]: might need to search all files for the `//`

**Comment by ivorb** [comment]
There are a ton of media files here, are they all necessary? for example, I see 38 images linked in 04, but more than 38 images in the media folder. Fine if so, just want to verify they all should be there

File: Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md (hunk 1 above)
Comment (vulcanpanther) [reply Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md]: Thanks for spotting. I used CTRL+F and got some rogue // in there :) Fixed!

**Comment by vulcanpanther** [comment]
> There are a ton of media files here, are they all necessary? for example, I see 38 images linked in 04, but more than 38 images in the media folder. Fine if so, just want to verify they all should be there @vulcanpanther

Looks like all files were copied over, so I've now removed all the unnecessary ones 👍

**Comment by ivorb** [comment]
Looks like we might have a similar problem for 08 - can you confirm all those media files are necessary?

**Comment by vulcanpanther** [comment]
> Looks like we might have a similar problem for 08 - can you confirm all those media files are necessary?

You're right! They are indeed rogue images. I've removed those as well now :)

**Pull request merged**